dataplane/go.mod

```diff
@@ -14,13 +13,14 @@ require (
 	github.com/onsi/gomega v1.7.0
 	github.com/opentracing/opentracing-go v1.1.0
 	github.com/rs/xid v1.2.1
-	github.com/satori/go.uuid v1.2.0
 	github.com/sirupsen/logrus v1.4.2
 	github.com/vishvananda/netlink v1.0.0
 	github.com/vishvananda/netns v0.0.0-20190625233234-7109fa855b0f
 	google.golang.org/grpc v1.23.1
 )
 
+replace github.com/census-instrumentation/opencensus-proto v0.1.0-0.20181214143942-ba49f56771b8 => github.com/census-instrumentation/opencensus-proto v0.0.3-0.20181214143942-ba49f56771b8
```

Comment:
Why isn't this part of the common replace block?  Why do we have a replace for OpenCencus?

Reply:
According to the style introduces by Artem, we keep replaces to local modules and other replaces in different blocks. 
New go 1.13 fails during build, because OpenCensus has invalid version tag - `v0.1.0`, that's why we have to use replace